backend/lib/seeds/test_homes.csv

```diff
@@ -1,103 +0,0 @@
-street_number,street_name,unit_number,city,state,zip_code,building_type,assignment_group,visit_order
```

Comment:
This & `test_seeds` were removed in https://github.com/codeforboston/urban-league-heat-pump-accelerator/commit/f097fa46ca78a8bec6d85c6f449b6a0ab36f17f2